util/transformer.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+//
+// Created by amir on 23/12/2020.
+//
```

Comment:
I'd delete this to save space. git history keeps an accurate account of who created what